.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 orbs:
   slack: circleci/slack@2.5.0
-  cla-end-to-end-tests: ministryofjustice/cla-end-to-end-tests@volatile
+  cla-end-to-end-tests: ministryofjustice/cla-end-to-end-tests@dev:alpha
```

Comment:
Please can you revert back to `@volatile`: 

`@volatile` is a special tag that points to the most recent version of the orb.